### PR TITLE
Update clippy.json

### PR DESCRIPTION
**Pull request by human-being-1**

### CODE REVIEW

File: clippy.json [hunk 1]
```diff
@@ -53,6 +56,12 @@
...
-    "gmail.com": "Do you need glasses?",
-    "mail.google.com": "Do you need glasses?"
+    "gmail.com": [
+        "Do you need glasses?",
+        "Maybe try Outlook? wink wink..."
+    ],
+    "mail.google.com": [
+        "Do you need glasses?",
+        "Maybe try Outlook? wink wink..."
+    ],
```
Comment (capJavert) [inline clippy.json]: This comma is extraneous and file is not JSON valid. Pls adjust and you can check validity of your JSON here https://jsonlint.com/

File: clippy.json [hunk 2]
```diff
@@ -36,7 +36,10 @@
         "I won't watch...",
         "Do you need recommendations?"
     ],
-    "live.com": "I am homeeee",
+    "live.com": [
+        "I am homeeee",
+        "It looks like you're on Outlook. Would you like help?"
```
Comment (capJavert) [inline clippy.json]: It looks like you're on Outlook. Would you like **some** help? sounds better to me.

**Review by capJavert — CHANGES_REQUESTED** [review]

**Comment by capJavert** [comment]
@NanhuGITHUB tnx for contribution, pls address issues from the review and then we can merge this!

**Comment by human-being-1** [comment]
Okay

File: clippy.json (hunk 1 above)
Comment (human-being-1) [reply clippy.json]: It was copy pasting

Comment (human-being-1) [reply clippy.json]: I did the necessary changes

File: clippy.json (hunk 2 above)
Comment (human-being-1) [reply clippy.json]: I changed it.

**Comment by human-being-1** [comment]
I did the changes.

**Review by capJavert — APPROVED** [review]
🤘

**Pull request merged**